docs/src/capi/howto/external-etcd.md

```diff
@@ -0,0 +1,127 @@
+# CAPI External Etcd
+
+To replace the built-in [dqlite][dqlite] database with an external etcd to
+manage the kubernetes state in the CAPI workload cluster follow this
+`how-to guide`.
+This example shows how to create a 3-node workload cluster with an external
+etcd. 
+
+## Prerequisites
+
+To follow this guide, you will need:
+
+- Clusterctl
+- A CAPI management cluster initialized with the infrastructure, bootstrap and
+  control plane providers of your choice
+- Secured 3-node etcd deployment
+
+Please refer to the [getting-started][getting-started] for instructions.
+
+## Create Kubernetes Secrets
+
+Create three kubernetes secrets:
+
+- {cluster-name}-etcd-servers
+- {cluster-name}-etcd
+- {cluster-name}-apiserver-etcd-client
+
+```{note}
+Replace {cluster-name} with the name of your cluster.
+It is important to follow this naming convention for the secrets since the providers will be looking for these names.
+Please note that this example uses `c1` for the cluster name.
+```
+
+Create the secret for the etcd servers:
+
+```
+kubectl apply -f - <<EOF 
+apiVersion: v1
+kind: Secret
+metadata:
+  name: c1-etcd-servers
+  namespace: default
+
+data:
+  servers: $(echo -n "https://etcd-1:2379,https://etcd-2:2379,https://etcd-3:2379" | base64 -w0)
+EOF
+```
+
+To export the path to your etcd certs directory, use this command:
+
+```
+export CERTS_DIR=path/to/etcd-certs
+```
+
+Replace /path/to/etcd-certs with the actual path where you generated or stored
+your etcd certificates.
+
+Create the secret for the etcd root ca:
+
+```
+kubectl create secret generic c1-etcd \
+  --from-file=tls.crt="$CERTS_DIR/etcd-root-ca.pem"
```

Comment:
```suggestion
kubectl create secret generic c1-etcd \
  --from-file=tls.crt="<path-to-etcd-ca-crt>"
```
Should we just drop the `$CERTS_DIR` bits and do this?

Reply:
I prefer the variable method, otherwise the user has to copy/paste and edit the commands and also you need to explain what needs replacing